angularjs-portal-home/src/main/webapp/my-app/layout/partials/home-toggle.html

```diff
@@ -1,22 +1,22 @@
 <span class="layout-chooser" ng-if="toggle" ng-class="{ 'hidden-xs' : !$storage.mobileWidgetToggle}">
-  <li ng-class="{active : modeIs('list')}">
-    <a  href="javascript:;" ng-click="switchMode('list')"
-      tooltip-html-unsafe="Icons"
+  <li ng-class="{active : modeIs('compact')}">
```

Comment:
Would making this `{active : !modeIs('expanded')}` catch any edge cases where mode is somehow neither `compact` nor `expanded`?